lib/find_chrome.js

```diff
@@ -215,14 +217,94 @@ function findChromeExecutables(folder) {
   return installations;
 }
 
-function installations() {
+/**
+ * @param {!Array<string>} configurations
+ * @return {!Promise<!Array<string>>}
+ */
+async function installations(configurations) {
+  if (configurations[0] === 'chromium')
```

Comment:
For newest browser features, I'd like to be able to say ['canary', 'chromium'] so that if there was canary, i would use it, otherwise i would get chromium.
```suggestion
  const configs = new Set(configurations);
  // always prefer stable
  if ('stable' in configs || '*' in configs) {
    path = darwin();
    if (path) return path;
  }
  // always prefer canary after stable
  if ('canary' in configs || '*' in configs) {
    path = darwin(true);
    if (path) return path;
  }
  // always prefer puppeteer revision of chromium
  if ('chromium' in configs || '*' in configs)
    return downloadChromium(puppeteerVersion);

  // end up downloading specified revision
  for (const c of configs)
    if (c.startsWith('r')) return downloadChromium(c);    
```